activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -394,7 +394,7 @@ def load_schema(db_config, format = ActiveRecord.schema_format, file = nil) # :n
         Migration.verbose = verbose_was
       end
 
-      def schema_up_to_date?(configuration, format = ActiveRecord.schema_format, file = nil)
```

Comment:
This changes the method signature of a public method. It's probably better to rename `format` to `_` to explicitly mark it as unused, and submit a separate PR to remove the parameter as a potentially-breaking change.

Reply:
Ah okay, that makes sense, wasn't something I'd considered. I've done that. Looks like it isn't actually used even before this PR.